api/nodes/serializers.py

```diff
@@ -116,6 +121,19 @@ def get_registration_count(self, obj):
     def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
+    def get_unread_comments_count(self, obj):
```

Comment:
I went ahead and created a project (shared between two users) with unread comments for each user. But I didn't see the `unread_comments_count` field appear on the Node list or Detail pages at all. Am I missing the goal of this feature?